Automattic/woocommerce-payments
# Fix #5963. GPay shows $0 for subscription_variation products

Fixes #5963

#### Changes proposed in this Pull Request




This change takes into account `subscription_variation` when re-adding products to the cart in the `ajax_add_to_cart()` function.


A similar fix was applied on the function `WC_Payments_Payment_Request_Button_Handler::get_product_price()` on the same file.



#### Testing instructions


1. Go to the product page of a subscription. Make sure the product type is subscription_variation and has free shipping
2. Click on Google Pay button
3. Check the totals, should have the Product price
4. Select a shipping method
5. Check the totals again, the product price is $0




<img width="465" alt="Screen Shot 2023-04-03 at 13 41 58" src="https://user-images.githubusercontent.com/1821060/229830982-3350a5dc-2bd5-4cd1-9846-523ae46d09aa.png">

<img width="352" alt="Screen Shot 2023-04-03 at 14 10 52" src="https://user-images.githubusercontent.com/1821060/229831009-e77eadc9-9409-466c-9626-5256b13540d6.png">


-------------------


**Post merge**
